litmus-portal/frontend/src/graphql/index.ts

```diff
@@ -57,16 +57,23 @@ export const GET_USER = gql`
       username
       email
       name
-      project_id
-    }
-  }
-`;
-
-export const GET_PROJECT = gql`
-  query getProject($projectID: String!) {
-    getProject(projectID: $projectID) {
-      name
-      id
+      projects {
```

Comment:
@ishangupta-ds You took projects, please make it project.

Reply:
It is `projects`, it has been updated in the server